### PR TITLE
docs(pie-docs): DSW-1483 add code docs page for pie-card

### PR DESCRIPTION
**Pull request by xander-marjoram**
## Describe your changes (can list changeset entries if preferable)
- Add `/components/card/code/` page

## Author Checklist (complete before requesting a review)
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests
- [ ] If it is a `PIE Docs` change, I have reviewed the Docs site preview
- [ ] If it is a component change, I have reviewed the Storybook preview
- [ ] If there are visual test updates, I have reviewed them properly before approving

### Reviewer 1
- [ ] If it is a `PIE Docs` change, I have reviewed the PR preview
- [ ] If there are visual test updates, I have reviewed them

### Reviewer 2
- [ ] If it is a `PIE Docs` change, I have reviewed the PR preview
- [ ] If there are visual test updates, I have reviewed them

### CODE REVIEW

File: apps/pie-docs/src/_11ty/shortcodes/componentDetailsTable.js [hunk 1]
```diff
@@ -5,6 +5,10 @@
  * @returns {string} - The HTML representation of a table row.
  */
 const buildRow = (cells) => cells.map((cell) => {
+    if (cell === null) {
+        return '<td></td>';
```
Comment (xander-marjoram) [inline apps/pie-docs/src/_11ty/shortcodes/componentDetailsTable.js:9]: This means we can have an empty cell if we want, like for a string field it doesn't make sense (to me) to say what the "options" are.

Comment (xander-marjoram) [reply apps/pie-docs/src/_11ty/shortcodes/componentDetailsTable.js:9]: Compare [this props table](https://pr1002-docs.pie.design/components/card/code/#props) with the bottom half of [button's props table](https://pie.design/components/button/code/#props)

**Review by raoufswe — DISMISSED** [review]

File: apps/pie-docs/src/_11ty/shortcodes/componentDetailsTable.js (hunk 1 above)
Comment (jamieomaguire) [reply apps/pie-docs/src/_11ty/shortcodes/componentDetailsTable.js:9]: Screenshot 2023-11-21 at 12 23 34

Not sure if related but I'm not sure what I would do with this `aria` prop as a consumer

**Review by jamieomaguire — DISMISSED** [review]

File: apps/pie-docs/src/_11ty/shortcodes/componentDetailsTable.js (hunk 1 above)
Comment (xander-marjoram) [reply apps/pie-docs/src/_11ty/shortcodes/componentDetailsTable.js:9]: It's a fair point, I think at the moment it only has an optional label property. I'll give it a go

**Review by fernandofranca — DISMISSED** [review]

**Pull request merged**